### ISSUE TITLE
Seal saying my version of youtube is outdated when its not.

### ISSUE THREAD

**Issue by TeamDraft** [JunkFood02/Seal#1620]
### Checklist

- [ ] I'm reporting a bug unrelated to a specific site.
- [X] I've verified that I'm running the [**latest version**](https://github.com/yt-dlp/yt-dlp/releases/latest) of yt-dlp.
- [X] I've verified that I'm running the latest [**stable version**](https://github.com/JunkFood02/Seal/releases/latest/) of Seal or any later [**preview versions**](https://github.com/JunkFood02/Seal/releases).
- [X] I've read the [**Contributing guidelines**](https://github.com/JunkFood02/Seal/blob/main/CONTRIBUTING.md) and [**Code Of Conduct.**](https://github.com/JunkFood02/Seal/blob/main/CODE_OF_CONDUCT.md)
- [X] I've checked that the site i'm trying to download from is in the [**Supported Sites**](https://github.com/yt-dlp/yt-dlp/blob/master/supportedsites.md) list from yt-dlp
- [X] I understand that the issue will be (ignored/closed) if I intentionally remove or skip any mandatory field.

### Describe the bug

As it says in the title. Immediately cancels download saying I have a outdated version of youtube installed but the play store says otherwise. 

### To Reproduce

1.) Enter playlist information for download, then confirm
2.) Error occurs

### Error reports

App version: 1.12.1 (11212)
Device information: Android 12 (API 31)
Supported ABIs: [arm64-v8a, armeabi-v7a, armeabi]
Yt-dlp version: 2024.08.06

URL: https://www.youtube.com/watch?v=wNvOUkRTkz8
ERROR: [youtube] yuFI5KSPAt4: The following content is not available on this app.. Watch on the latest version of YouTube.

### Screenshots & Screen Records

![Screenshot_20240807-115503_Seal](https://github.com/user-attachments/assets/09af360d-994d-43a3-89da-491b62224866)

**Comment by Ehab-d** [comment]
This is due YouTube restrictions. The app is not talking about the YouTube on your phone anyway.

**Comment by JunkFood02** [comment]
Duplicate of https://github.com/yt-dlp/yt-dlp/issues/10108

**Comment by TeamDraft** [comment]
Thanks for the info! I also found out it was due to a security suite I had
installed. Even though the vpn wast active itself, it had a built in app
proxy feature that gives apps there own personal vpn IP Client for secure
use outside the dedicated VPN Network (IP Masking, not a true VPN.) I
didn't have the option switched on for it to be used with a proxy and
therefore caused this error I believe. I apologize to everyone in the
community if I caused a false flag report. Thank you for the help. I also
appreciate you junkfood02 for app.
